Sources/StreamChat/APIClient/Endpoints/Payloads/MessagePayloads.swift

```diff
@@ -56,6 +56,8 @@ struct MessageSearchResultsPayload: Decodable {
 /// An object describing the incoming message JSON payload.
 class MessagePayload: Decodable {
     let id: String
+    /// Only messages from `translate` endpoint contain `cid`
+    let cid: ChannelId?
```

Comment:
This comment has a high chance of being outdated in the future, I would consider discarding this comment